### ISSUE TITLE
//examples/testing fails on Windows

### ISSUE THREAD

**Issue by filipesilva** [bazelbuild/rules_typescript#272]
Trying to run the chromium test will fail under windows, failing to capture ChromeHeadless.

```
kamik@RED-X1C6 MINGW64 /d/work/rules_typescript (webtesting-windows)
$ bazel test //examples/testing:testing_chromium-local
INFO: Build options have changed, discarding analysis cache.
INFO: Analysed target //examples/testing:testing_chromium-local (53 packages loaded).
INFO: Found 1 test target...
FAIL: //examples/testing:testing_chromium-local (see C:/users/kamik/_bazel_kamik/5c2rzegw/execroot/build_bazel_rules_typescript/bazel-out/x64_windows-fastbuild/testlogs/examples/testing/testing_chromium-local/test.log)
INFO: From Testing //examples/testing:testing_chromium-local:
==================== Test output for //examples/testing:testing_chromium-local:

(very long PATH related output omitted for brevity)

WARNING: source-map-support module not installed.
    Stack traces from languages like TypeScript will point to generated .js files.
    Set install_source_map_support = False in @build_bazel_rules_typescript//internal/karma:karma_bin_bin to turn off this warning.

14 09 2018 16:20:17.776:INFO [karma]: Karma v1.7.1 server started at http://0.0.0.0:9876/
14 09 2018 16:20:17.778:INFO [launcher]: Launching browser ChromeHeadless with unlimited concurrency
14 09 2018 16:20:17.783:INFO [launcher]: Starting browser ChromeHeadless
14 09 2018 16:21:17.785:WARN [launcher]: ChromeHeadless have not captured in 60000 ms, killing.
14 09 2018 16:21:19.787:WARN [launcher]: ChromeHeadless was not killed in 2000 ms, sending SIGKILL.
14 09 2018 16:21:21.788:WARN [launcher]: ChromeHeadless was not killed by SIGKILL in 2000 ms, continuing.
================================================================================
Target //examples/testing:testing_chromium-local up-to-date:
  C:/users/kamik/_bazel_kamik/5c2rzegw/execroot/build_bazel_rules_typescript/bazel-out/x64_windows-fastbuild/bin/examples/testing/testing_chromium-local.gen.json
  C:/users/kamik/_bazel_kamik/5c2rzegw/execroot/build_bazel_rules_typescript/bazel-out/x64_windows-fastbuild/bin/examples/testing/testing_chromium-local
INFO: Elapsed time: 97.057s, Critical Path: 87.35s
INFO: 1 process: 1 local.
INFO: Build completed, 1 test FAILED, 4 total actions
//examples/testing:testing_chromium-local                                FAILED in 87.0s
  C:/users/kamik/_bazel_kamik/5c2rzegw/execroot/build_bazel_rules_typescript/bazel-out/x64_windows-fastbuild/testlogs/examples/testing/testing_chromium-local/test.log

INFO: Build completed, 1 test FAILED, 4 total actions
```

`bazel test //examples/testing:testing_firefox-local` will seem not to fail, but checking the test log reveals the following message:
```
exec ${PAGER:-/usr/bin/less} "$0" || exit 1
Executing tests from //examples/testing:testing_firefox-local
-----------------------------------------------------------------------------
This test always passes. Your test was not run.

This dummy test was inserted in place of the web test you intended
to run because:
The browser configuration you requested is temporarily disabled.

Disabled browser: @io_bazel_rules_webtesting//browsers:firefox-local

Why was this browser disabled?
firefox not supported on windows
```
